### PR TITLE
link fixes

### PR DESCRIPTION
**Pull request by bmicklea**
Updated anchor links (broken) and clarified the downloads.

### CODE REVIEW

File: docs/src/docs/cli/installation.md [hunk 1]
```diff
@@ -53,25 +53,25 @@ That's it!
 
 Make sure you get the correct download for your hardware.
 
-1. <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-windows-x86_64.zip"
+1. Windows: <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-windows-arm64.zip"
```
Comment (amisevsk) [inline docs/src/docs/cli/installation.md:56]: We probably want x86 to be first for Windows. There are not many arm machines that run windows normally.

File: docs/src/docs/cli/installation.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ That's it!
 
 Make sure you get the correct download for your hardware.
 
-1. <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-linux-arm64.tar.gz"
+1. Linux: <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-linux-arm64.tar.gz"
```
Comment (amisevsk) [inline docs/src/docs/cli/installation.md:103]: Similar to windows, 90%+ of linux users are going to be using x86_64.

**Review by amisevsk — APPROVED** [review]

File: docs/src/docs/cli/installation.md (hunk 1 above)
Comment (bmicklea) [reply docs/src/docs/cli/installation.md]: fixed

File: docs/src/docs/cli/installation.md (hunk 2 above)
Comment (bmicklea) [reply docs/src/docs/cli/installation.md]: fixed

**Pull request merged**